mediator/med_phases_prep_ocn_mod.F90

```diff
@@ -161,7 +161,7 @@ subroutine med_phases_prep_ocn_merge(gcomp, rc)
        end if
 
        ! custom merges to ocean
-       if (trim(coupling_mode) == 'cesm' .or. trim(coupling_mode) == 'hafs') then
+       if (trim(coupling_mode) == 'cesm') then
```

Comment:
check all allowed  values and error out if it is none of them.

Reply:
I think it is still safe. Also, this was creating a conflict and I solve it via interface but I am still seeing the old version. So, I am not sure how can I add extra debug in here. Let me know, if you have idea.